public/css/index.css

```diff
@@ -0,0 +1,5 @@
+#logo {
```

Comment:
This is the same selector (`#logo`) than in `main.css`, this can lead to various debugging headaches.
You should probably use two different classes for the smaller and bigger version of the logo.